docs/code/MlNetCookBook.md

```diff
@@ -1022,3 +1023,49 @@ ITransformer loadedModel;
 using (var fs = File.OpenRead(modelPath))
     loadedModel = newContext.Model.Load(fs);
 ```
+
+## How can I read and write binary data?
+Other than using text files, ML.NET will allow you to read and write binary data. This has a few advantages such as not having to specify a schema, can improve reading times, and are generally smaller than text files.
+
+To write binary data you need some data to be able to save. Specifically you need an instance of an `IDavaView`. Below is a code snippet that uses the iris data as an example.
+
+```csharp
+// Data model for the iris data
+public class IrisData
+{
+    public float Label { get; set; }
+    public float SepalLength { get; set; }
+    public float SepalWidth { get; set; }
+    public float PetalLength { get; set; }
+    public float PetalWidth { get; set; }
+}
+
+// An array of iris data points
+var dataArray = new[]
+{
+    new IrisData { Label=1, PetalLength=1, SepalLength=1, PetalWidth=1, SepalWidth=1 },
+    new IrisData { Label=0, PetalLength=2, SepalLength=2, PetalWidth=2, SepalWidth=2 }
+};
+
+// Create the ML.NET context.
+var context = new MLContext();
+
+// Create the data view from an IEnumerable.
+// This method will use the definition of IrisData to understand what columns there are 
+// in the data view. However, the objects in ML.NET are only "promises" of data since 
+// ML.NET operations are lazy. One way to get a look at the data is with Schema Comprehension.
+// Refer to this document for more information - https://github.com/dotnet/machinelearning/blob/master/docs/code/SchemaComprehension.md
+var data = context.Data.LoadFromEnumerable(dataArray);
```

Comment:
I think that in the cookbook we tend to have high level explanation. Since we already discuss in other places how to create an `IDataView` (from enumerable, by loading test data...) I would suggest to skip this part and write a sentence like the following:

Assume that we already have an `IDataView`, which could be the result of loading text data, or the result of some transformation.  

If you see the section for saving and loading a model, it's pretty short and we don't go into how to generate the model in that section.